src/renderers/webgl/WebGLTextures.js

```diff
@@ -793,7 +793,17 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 		} else if ( texture.isFramebufferTexture ) {
 
-			// texture data extracted from framebuffers require mutuable textures defined via gl.copyTexImage2D()
+			if ( useTexStorage && allocateMemory ) {
+
+				const mips = ( texture.minFilter !== NearestFilter && texture.minFilter !== LinearFilter ) ? Math.log2( Math.max( image.width, image.height ) ) + 1 : 1;
+
+				state.texStorage2D( _gl.TEXTURE_2D, mips, glInternalFormat, image.width, image.height );
+
+			} else {
+
+				state.texImage2D( _gl.TEXTURE_2D, 0, glInternalFormat, image.width, image.height, 0, glFormat, glType, null );
+
+			}
```

Comment:
I've updated this code section similar to the other ones in `uploadTexture()`. Otherwise `FramebufferTexture` is unusable with WebGL 1.

Reply:
Also made sure that `getMipLevels()` is used.

Reply:
Thanks for the updates! However, I don't  believe we can use `getMipLevels()` as is because for that to return  the right value, `textureNeedsGenerateMipmaps` must be true, which is  false if  `generateMipmaps`  is false. That is correctly false in  this case because they're not being auto-generated, but manually filled in.

Reply:
I have enhanced the logic in `getMipLevels()` and added a special handling for `FramebufferTexture`. This should ensure things work as before.

Reply:
Perfect, thanks!